### DISCUSSION TITLE
Antd Overrides my css module style

### DISCUSSION THREAD

**Discussion by fayaz-pn** [ant-design/ant-design#42040] [Q&A]
When I create custom style in my component.module.css
AntDesign overrides it with classes such as 
`:where(.css-dev-only-do-not-override-13ifdra).ant-layout .ant-layout-side`

I haven't done any set up yet such as importing AntDesign style sheet to my main style sheet. 
Im using Nextjs with Antdesign

**Comment by zombieJ** [comment]
You can raise select priority to override antd default style:

```css
#root .myCls {
  // Your style here
}
```

**Reply by fayaz-pn** [reply]
![image](https://user-images.githubusercontent.com/130346787/234790752-52f45d38-de66-409b-a00f-b93b34dbae0a.png)


Sorry, no change. 

When I used antDesign with tailwind, I encountered the same problem where antD overrides tailwind styles 
I could just edit the post-css config and would fix it

But here Im using css modules and I wondering if there is any solution like that ?

**Reply by fayaz-pn** [reply]
![image](https://user-images.githubusercontent.com/130346787/234791630-57e8f490-c350-461a-8e31-e457ed3cd970.png)
This is how my dev console is. 

I haven't done any configuration as of now. Should I do some config to fix this ?

**Reply by osmolyar** [reply]
@fayaz-pn the issue isn't the 'overriding' with the :where(.css-dev-only-do-not-override-13ifdra); if you were using a regular .css file (not module.css) then the :where clause would be lower priority and your class styles would still work.  

However because you are using module.css, you would have to put the .ant-layout and .ant-layout-sider in a global.less file in order for them to apply - they won't apply from the module.css file because the classes are not defined in that file hence they aren't governed by the module logic that modifies the classes with a dynamic suffix.   

The downside is that if you put the styles in a global file then all your ant styles will get overwritten,  not just for the target page. So then you have to add another custom class to some container of the ant element being overridden and use that in your global.css style definition to make it specific to one page.
